pytest/lib/cluster.py

```diff
@@ -379,17 +381,15 @@ def start(self, *, boot_node: BootNode = None, skip_starting_proxy=False):
         env["RUST_LOG"] = "actix_web=warn,mio=warn,tokio_util=warn,actix_server=warn,actix_http=warn," + env.get(
             "RUST_LOG", "debug")
 
-        self.stdout_name = os.path.join(self.node_dir, 'stdout')
-        self.stderr_name = os.path.join(self.node_dir, 'stderr')
-        self.stdout = open(self.stdout_name, 'a')
-        self.stderr = open(self.stderr_name, 'a')
+        self._stdout = open(os.path.join(self.node_dir, 'stdout'), 'a')
```

Comment:
'a' or 'ab'?

Reply:
In the end it doesn’t matter. The file is passed to the subprocess.